server/monitor/src/main/java/org/apache/accumulo/monitor/EmbeddedWebServer.java

```diff
@@ -118,19 +98,17 @@ public void start() {
     }
   }
 
-  public void stop() {
+  private void stop() {
```

Comment:
sgtm. Long-term we should be moving to treating the objects that can be started/stopped with some sort of Service interface abstraction (e.g. TabletServer, Monitor, GC), but doesn't need to happen here and such a refactoring could invalidate anyways.